js/views/playingScreen.js

```diff
@@ -19,11 +19,15 @@ var PlayingScreen = React.createClass({
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
     this.browserSupportsTouch = this.props.controller.state.browserSupportsTouch;
-    this.isVideo360 = this.props.controller.state.isVideo360;
+    this.videoVr = this.props.controller.videoVr;
 
     return {
       controlBarVisible: true,
-      timer: null
+      timer: null,
+      isMouseDown: false,
+      isMouseMove: false,
+      XMouseStart: 0,
+      YMouseStart: 0,
```

Comment:
Can we rename these 4 states to include "vr" since setting them requires videoVr to be true (such as in the handlePlayerMouseDown function)? Or otherwise, we can set them regardless of whether videoVr is true or not and adjust logic accordingly.

Reply:
\+